fhservices.cfg.j2

```diff
@@ -10,6 +10,12 @@ define command {
        command_line   /opt/rhmap/nagios/plugins/fh-check-component-health.sh $ARG1$ 8080
 }
 
+# Check for valid response from Memcached
+define command {
+       command_name   check_memcached
+       command_line   $USER1$/check_tcp -H memcached -p $ARG1$ -E -s 'version\n' -e 'VERSION' -w2 -c5 -t5
```

Comment:
@rhcarvalho In our SaaS grids there were some issues around the version of memcached being used so by passing in these version arguments we're getting the version being used, may not be relevant for On-Prem but dont see any harm in leaving it there.

The thresholds are:
-w <warning-response-time>
-c <critical-response-time>
-t <timeout>

Each of these are measured in seconds.